mid2low_v: 0.46
low2mid_v: 0.55
mid2low_omega: 1.0
low2mid_omega: 1.4

high2mid_v: 2.6
mid2high_v: 3.0
high2mid_omega: 4.0
mid2high_omega: 6.0

k_low: 0.45
k_mid: 2.3
k_high: 2.9
